kernel/aster-nix/src/fs/ext2/inode.rs

```diff
@@ -776,6 +813,8 @@ impl Inner {
 
     pub fn remove_entry(&mut self, name: &str, offset: usize) -> Result<()> {
         let entry = DirEntryWriter::new(&self.page_cache, offset).remove_entry(name)?;
+        self.inode_impl
+            .set_mtime(self.fs().time_provider().current());
```

Comment:
According to Table 15-2, unlink should update both the mtime and ctime of the dir. And the `ctime` of the deleted file if its link count is greater than 1.